swagger2.0/oic.r.rule.action.swagger.json

```diff
@@ -0,0 +1,191 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "OCF Rule Action",
+    "version": "20190910",
+    "license": {
+      "name": "OCF Data Model License",
+      "url": "https://github.com/openconnectivityfoundation/core/blob/e28a9e0a92e17042ba3e83661e4c0fbce8bdc4ba/LICENSE.md",
+      "x-copyright": "copyright 2019 Open Connectivity Foundation, Inc. All rights reserved."
+    },
+    "termsOfService": "https://openconnectivityfoundation.github.io/core/DISCLAIMER.md"
+  },
+  "schemes": ["http"],
+  "consumes": ["application/json"],
+  "produces": ["application/json"],
+  "paths": {
+    "/RuleActionResURI": {
+      "get": {
+        "description": "Rule Action contains a link to Scene Collection and a value to be set for the \"lastScene\" Property within that Collection.",
+        "parameters": [
+          {"$ref": "#/parameters/interface-rw"}
+        ],
+        "responses": {
+          "200": {
+            "description" : "Retrieves the Rule action.",
+            "x-example":
+            {
+              "rt": ["oic.r.rule.action"],
+              "if": ["oic.if.rw", "oic.if.baseline"],
+              "link": [{"href": "/myscene","rt":["oic.wk.scenecollection"],"if":["oic.if.a","oic.if.ll","oic.if.baseline"]}],
+              "lastscene": "desiredlastscenevalue"
+            },
+            "schema": { "$ref": "#/definitions/ruleaction" }
+          }
+        }
+      },
+      "post": {
+        "description": "",
+        "parameters": [
+          {"$ref": "#/parameters/interface-rw"},
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": { "$ref": "#/definitions/ruleaction" },
```

Comment:
definitions/ruleaction has both "link" and "scenevalue" in the "required" tag.  I assume that an UPDATE request does not have to include both.  In fact, the example below only includes "scenevalue" (shown as "lastscene" below).  I think we need a separate schema for "post".